contracts/BasePaymaster.sol

```diff
@@ -26,37 +25,37 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
         return address(relayHub);
     }
 
-    // Gas stipends for acceptRelayedCall, preRelayedCall and postRelayedCall
-    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 50000;
-    uint256 constant private PRE_RELAYED_CALL_GAS_LIMIT = 100000;
-    uint256 constant private POST_RELAYED_CALL_GAS_LIMIT = 110000;
+    //overhead of forwarder verify+signature, plus hub overhead.
+    uint256 constant public FORWARDER_HUB_OVERHEAD = 50000;
+
+    //These parameters are documented in IPaymaster.GasLimits
+    uint256 constant public PRE_RELAYED_CALL_GAS_LIMIT = 100000;
+    uint256 constant public POST_RELAYED_CALL_GAS_LIMIT = 110000;
+    uint256 constant public PAYMASTER_PAYS_ABOVE = PRE_RELAYED_CALL_GAS_LIMIT + FORWARDER_HUB_OVERHEAD;
 
     function getGasLimits()
-    external
+    public
```

Comment:
changed to public to allow easy accessing from a subclass:
https://github.com/opengsn/gsn/pull/429/files#diff-77990bc92cd4dea5097e507e088f3048R92
```
IPaymaster.GasLimits private limits = super.getGasLimits();
```